definitions/ValuePoint.h

```diff
@@ -432,6 +432,28 @@ namespace Exchange {
             _adminLock.Unlock();
         }
 
+        inline void Updated() const
+        {
+            if (_clients.size() > 0) {
```

Comment:
And although 99% change having _clients.size() like this outside the lock like this is probably safe, I would just include it in the lock

Reply:
This has no use to place it within the lock. If the client subscribes between the check and the lock, the client still has the correct value, as after subscription the client always has to get the initial value which is already updated. So it should always start with the correct value regardless of this unlocked situation.

Reply:
That is not why I made the remark :) Getting the size might potentially be unsafe and crash (if the clients would be updated at the same time) depending on how it is implemented. As it is a vector big chance it indeed is just getting a cached value, but in that case it is so quick it does not hurt to get it into the lock, so why take the chance :) But that is why it is only a comment, not a review remark, happy to leave it as it is

Reply:
The crash agree, that is something to consider, did not think about a potential crash (implementation of Size())....